ansible/configs/sap-integration/files/pv-registry.j2

```diff
@@ -0,0 +1,14 @@
+apiVersion: v1
```

Comment:
what are you using this for?

Where is `nfs_server_address` defined?

With the version of OpenShift you are using, you don't need this when using IPI.

Reply:
done